app/customer.py

```diff
@@ -0,0 +1,172 @@
+import datetime
+import json
+from typing import Union, List
+from app.shop import Shop
+from app.car import get_list_nearest_shops
+
+
+class Customer:
+    name: str
+    product_cart: dict
+    location: list

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]